.github/workflows/slack-office-hours-design.yml

```diff
@@ -0,0 +1,101 @@
+name: Slack Office Hours Reminder
+
+on:
+  schedule:
+    # Run every Monday at 09:00AM CT, 02:00PM UTC
+    - cron: '0 14 * * MON'
+
+jobs:
+  post-reminder:
+    name: Post design reminder to carbon-design-system and carbon-announcements
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send custom JSON data to Slack workflow
+        id: slack
+        uses: slackapi/slack-github-action@v1.22.0
+        with:
+          payload: |
+            {
+              "username": "Carbon Design System",
+              "icon_url": "https://user-images.githubusercontent.com/3360588/192045905-5d9705af-92e2-4432-805e-15db98571e8b.png",
+              "channel": ["#carbon-design-system", "#carbon-announcements"],
+              "text": "Join us for Carbon Design Office Hours!",
+              blocks: [
+                {
+                  type: "header",
+                  text: {
+                    type: "plain_text",
+                    text: "Join us for Carbon Design Office Hours!",
+                    emoji: true,
+                  },
+                },
+                {
+                  type: "divider",
+                },
+                {
+                  type: "image",
+                  image_url:
+                    "https://carbondesignsystem.com/static/406b7e7bf6865c094dd9a7a0bcb8bd41/3cbba/light-getting-started-developers.png",
```

Comment:
@thyhmdo when you have the new image for this, just put it in this comment thread and we can use the generated link github provides

Reply:
@laurenmrice let me know if you have any feedback on this :) 

![Carbon-Announcement](https://user-images.githubusercontent.com/77697351/207710625-c246e20f-90af-43e4-84a7-c35f75005503.jpg)

Reply:
Updated with new link 👍